src/components/markdown.js

```diff
@@ -61,7 +73,7 @@ const Markdown = props => {
     if (props.username) {
       return (
         <>
-          {`<a href="${props.base}/${props.username}" target="blank"><img align="center" src="${props.icon}" alt="${props.username}" height="30" width="30" /></a>`}
+          {`<a href="${props.base}/${props.username}" target="blank" style="margin:0.5rem"><img align="center" src="${props.icon}" alt="${props.username}" height="30" width="30" /></a>`}
```

Comment:
changing `width="30"` to `width="40"` will add a good margin between icons

Reply:
Alright will do that :+1:

Reply:
I've added a preview for both the configurations in the comment down below, which one should I proceed with ?

Reply:
Please see the comment below